x/cwgrant/types/cw.go

```diff
@@ -0,0 +1,83 @@
+package types
+
+import (
+	"fmt"
+
+	"github.com/CosmWasm/wasmd/x/wasm/types"
+	wasmVmTypes "github.com/CosmWasm/wasmvm/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/gogoproto/proto"
+)
+
+// SudoMsg defines the message sudo enum that is sent to the CosmWasm contract.
+type SudoMsg struct {
+	// CWGrant defines the enum variant of the grant message.
+	CWGrant *CWGrant `json:"cw_grant"`
+}
+
+// CWGrant defines the CWGrant variant of the SudoMsg.
+type CWGrant struct {
```

Comment:
do we think this info is enough for a contract to decide if to accept the fee grant request or not?

Reply:
Yea, i think it should be